doc/nrf/shortcuts.txt

```diff
@@ -228,3 +228,8 @@
 
 .. |migration_contact_devzone| replace:: While we strive to document all breaking changes, the migration guides might not include the detailed migration steps for your use case.
    If you need assistance, contact Nordic Semiconductor's technical support on `DevZone`_.
+
+.. |HID_note| replace:: Apart from HID samples, the |NCS| provides a complete reference application design of :term:`Human Interface Device (HID)`.
```

Comment:
Consider mentioning nRF Desktop in link name - e,g. `nrf_desktop_HID_ref` (this actually works as a reference to nRF Desktop application)